cli/src/commands/extensions/mod.rs

```diff
@@ -276,6 +276,7 @@ fn ask_permissions(extension: &Extension) -> Result<()> {
     print_permissions_list("Read", "path", permissions.read.get());
     print_permissions_list("Write", "path", permissions.write.get());
     print_permissions_list("Run", "command", permissions.run.get());
+    print_permissions_list("Run", "unsandboxed command", permissions.unsandboxed_run.get());
```

Comment:
Should we do something to make this stand out? Like make it print in red or something?

Reply:
All our permission requests are already somewhat highlighted in color. I feel like all of them should be read carefully anyway so there's no immediate **necessity** to highlight it even more. Though it is the most dangerous permission.